generated/models/inputs.go

```diff
@@ -85,6 +85,7 @@ func (i GetBooksInput) Validate() error {
 // GetBookByIDInput holds the input parameters for a getBookByID operation.
 type GetBookByIDInput struct {
 	BookID        int64
+	AuthorID      *string
```

Comment:
Right now if you specify `date-time` or `email`, the type is a special non-string type from `strfmt`, so I could see making this an `bson.ObjectId` being consistent with that behavior. I don't feel strongly, though. We could wait and see how people use it.